yb-voyager/cmd/common.go

```diff
@@ -1051,14 +1051,15 @@ func storeTableListInMSR(tableList []sqlname.NameTuple) error {
 
 // TODO: consider merging all unsupported field with single AssessmentReport struct member as AssessmentIssue
 type AssessmentReport struct {
-	VoyagerVersion      string                                `json:"VoyagerVersion"`
-	TargetDBVersion     *ybversion.YBVersion                  `json:"TargetDBVersion"`
-	MigrationComplexity string                                `json:"MigrationComplexity"`
-	SchemaSummary       utils.SchemaSummary                   `json:"SchemaSummary"`
-	Sizing              *migassessment.SizingAssessmentReport `json:"Sizing"`
-	Issues              []AssessmentIssue                     `json:"-"` // disabled in reports till corresponding UI changes are done(json and html reports)
-	TableIndexStats     *[]migassessment.TableIndexStats      `json:"TableIndexStats"`
-	Notes               []string                              `json:"Notes"`
+	VoyagerVersion                 string                                `json:"VoyagerVersion"`
+	TargetDBVersion                *ybversion.YBVersion                  `json:"TargetDBVersion"`
+	MigrationComplexity            string                                `json:"MigrationComplexity"`
+	MigrationComplexityExplanation string                                `json:"MigrationComplexityExplanation"`
```

Comment:
`MigrationComplexityExplaination`

Reply:
`Explanation` is the correct spelling  :)

Reply:
Oh right please update the PR description, The html report screenshot doens't reflect the current code's html

Reply:
yeah will do, those are old.